README.md

```diff
@@ -26,14 +26,7 @@ The health of a bot is the sum of its blocks' health.
 
 # Current state
 
-This project is currently an early prototype.
-Nothing is final and everything which has been implemented is subject to improvement.
-The current goal is to implement a basic version of as many of the features as possible.
-This allows design errors to be spotted early on,
-before any radical would be required when fixing them.
-This also means that all front-end development is kept to a bare minimum.
-At the moment the short-term goal is to make the already implemented features
-smoothly work in a networked environment.
+This project is currently an early prototype. Nothing is final, and everything which has been implemented is subject to improvement. The current goal is to achieve a basic version of as many of the features as possible. This allows design errors to be spotted early on before any radical would be required when fixing them. This also means that all front-end development is kept to a bare minimum. At the moment the short-term goal is to make the already implemented features smoothly work in a networked environment.
```

Comment:
You have removed the newlines only in a part of the `README.md` file. I do not agree with the change, but you weren't thorough nevertheless. I do not agree with the word change you have made: the meaning difference is not negligible in this context. I thank you for the rest of the changes - the insertion of two commas.

Reply:
Lets do it without the word replacement then?

Reply:
The only change I approve of is the addition of the two commas. In general, I would advise against doing any style changes: it's probably like that for a reason (there are exceptions of course).